components/brave_wallet_ui/components/desktop/views/accounts/accounts.tsx

```diff
@@ -82,7 +89,19 @@ export const Accounts = () => {
     [history]
   )
 
-  // memos
+  // memos && computed
+  const externalRewardsProvider =
+    externalRewardsInfo?.provider ?? undefined
+
+  const externalRewardsAccount =
+    getNormalizedExternalRewardsWallet(externalRewardsProvider)
```

Comment:
perf: consider inlining the value of `externalRewardsProvider` here

Reply:
Done